docs/company.md

```diff
@@ -0,0 +1,335 @@
+
+__Company__
+
+`GET /1/companies.json`
+Get a list of companies
+
+* Required permission: read
+* Arguments:
+    + `limit`: amount of results (default: 50)
+    + `page`: page to show (default: 1)
+* Sample response:
+
+````
+HTTP/1.1 200 OK
+{
+  "companies": [
+    {
+      "id": "42",
+      "name": "Magnetis",
+      "description": "Magnetis is the best online advisor ever",
+      "logo": "/companies/42/logo.png",
+      "website": "magnetis.com.br",
+      "social_links": [
+        "twitter.com/@magnetis"
+      ],
+      "markets": [
+        "financial services",
```

Comment:
@rafaelrinaldi 
Yes, there is a [Market](https://github.com/lucianot/dealbook2/pull/1/files#diff-6d222fa0a3349e108f4fabea665453ca) resource as well.

This is just the response format, but we don't need to store denormalized.